admin/class-digital-climate-strike-wordpress-admin.php

```diff
@@ -100,16 +104,16 @@ private function field_is_set($input, $field ) {
      */
     public function validate($input) {
         $valid = array();
-        $valid['show_digital_strike_widget'] = $this->field_is_set($input, 'show_digital_strike_widget') ? esc_attr($input['show_digital_strike_widget']) : 0;
+        $valid['show_digital_strike_widget'] = $this->field_is_set($input, 'show_digital_strike_widget') ? esc_attr($input['show_digital_strike_widget']) : self::DISABLE;
         $valid['language'] = $this->field_is_set($input, 'language') ? esc_attr($input['language']) : 'en';
-        $valid['cookie_expiration_days'] = $this->field_is_set($input, 'cookie_expiration_days') ? esc_attr($input['cookie_expiration_days']) : 1;
+        $valid['cookie_expiration_days'] = $this->field_is_set($input, 'cookie_expiration_days') ? esc_attr($input['cookie_expiration_days']) : self::ENABLE;
```

Comment:
I know it's an extra line of code, but I think this should have it's own constant for clarity. E.g. DEFAULT_COOKIE_EXPIRATION_DAYS

Reply:
Fixed.